src/cljs/athens/keybindings.cljs

```diff
@@ -50,19 +50,36 @@
            {:selection selection})))
 
 
+(defn filter-slash-options
+  [query]
+  (if (blank? query)
+    slash-options
+    (filter (fn [[text]]
+              (re-find (re-pattern (str "(?i)" query)) text))
+            slash-options)))
+
+
 (defn update-query
+  "3-arity is used during backspace.
```

Comment:
Do we need 3-arity? Could just pass empty string on backspace, and comment on it.